Fixes #3628 Remove memory check

gbraad opened the pull request:
**Fixes:** Issue #3628

## Solution/Idea

Removes the memory check, as this is incorrect for a 8GB macbook that is capable of running MicroShift (and podman).

## Proposed changes

Lowering the check makes this less relevant/meaningless; it says nothing about the ability in that case to run any of the supported presets. Any check needs to happen from the preset specific default values on startup, not installation.

1. Grab an 8GB macbook M1
2. Start the installer
3. ...

gbraad committed:
Fixes #3628 Remove memory check

Removes the memory check, as this is incorrect for a 8GB macbook that
is capable of running MicroShift. Lowering the check makes this less
relevant/meaningless; it says nothing about the ability in that case
to run any of the supported presets. Any check needs to happen from the
preset specific default values on startup, not installation.

praveenkumar commented on packaging/darwin/Distribution.in:
```diff
@@ -12,13 +12,6 @@
     <installation-check script="installCheck();"/>
     <script>
 function installCheck() {
-    if (9216000000 > system.sysctl('hw.memsize')) {
```
should we make it it 4GB instead remove it completely?

praveenkumar replied:
Looks like even 4G might not make sense if someone just want it to use for podman.

gbraad committed:
Remove PhysicalMemory condition for installation on Windows

Similarly to the macOS situation, the enforcement of the memory
requirements will happen on start of the VM. This will otherwise
prevent people to use Microshift or Podman preset on low-memory
machines.

gbraad commented:
As mentioned by @anjannath, these checks also exist for the MSI and have been removed as part of 
bc2324c. PTAL
